devtools/src/main/scala/kalix/devtools/impl/DockerComposeUtils.scala

```diff
@@ -107,13 +107,14 @@ case class DockerComposeUtils(file: String, envVar: Map[String, String]) {
 
   def userFunctionPort: Int =
     envVar
-      .get("USER_FUNCTION_PORT")
+      .get("USER_SERVICE_PORT")
+      .orElse(envVar.get("USER_FUNCTION_PORT")) // legacy name
```

Comment:
If USER_SERVICE_PORT is not an env var, we try USER_FUNCTION_PORT

Otherwise we read from file.